### PR TITLE
Remove specs-go/parsing APIs

### PR DESCRIPTION
**Pull request by bart0sh**
Got rid of using `parseQualifier` and `isLetter` APIs in version.go

### CODE REVIEW

File: specs-go/version.go [hunk 1]
```diff
@@ -183,23 +183,20 @@ func requiresV060(spec *Spec) bool {
 	}
 
 	// The v0.6.0 spec allows dots "." in Kind name label (class)
-	vendor, class := parseQualifier(spec.Kind)
-	if vendor != "" {
-		if strings.ContainsRune(class, '.') {
-			return true
-		}
+	if !strings.Contains(spec.Kind, "/") {
```
Comment (elezar) [inline specs-go/version.go:186]: There's a subtle difference here in that if there is no `/` the *class* is assumed to equal the Kind. and that would be checked for a `.`.

I don't think that's used in practice though.

Comment (elezar) [reply specs-go/version.go:186]: OK. Wait, I've just seen the other conditional below.

**Review by elezar — APPROVED** [review]
LGTM

**Comment by bart0sh** [comment]
@elezar Thank you for the fast review & merging!

**Pull request merged**